src/components/footer/__test__/index.test.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Footer from "../index";
+import { render, cleanup, fireEvent } from "../../../utils/test-utils";
+
+const setUp = (props = {}) => {
+  const component = render(<Footer {...props} />);
+  return component;
+};
+
+describe("Footer Component", () => {
+  let component;
+  beforeEach(() => {
+    component = setUp();
+  });
+
+  afterEach(cleanup);
+
+  it("should render the ac-logo without error", () => {
+    const logo = component.getAllByTestId("ac-logo");
+    expect(logo.length).toBe(1);
+  });
```

Comment:
To Daniel's point, there should be another test in here to account for the link. It would look something like this (also see comment about id name below - change it to "ac-footer-logo" in the component and in the above test.):
```
  it("should link to the home page", () => {
    const logo = component.getAllByTestId("ac-footer-logo");
    expect(logo).toHaveAttribute('href', '/');
  });
```